roles/friends_of_pul/tasks/main.yml

```diff
@@ -1,2 +1,7 @@
 ---
 # tasks file for roles/friends_of_pul
+- name: friends_of_pul | ensure memory limit is set in apache2 php.ini
```

Comment:
Could we do this in the PHP role instead? That way it will be available to all PHP applications, not just FPUL.

I'd put the task here: https://github.com/pulibrary/princeton_ansible/blob/main/roles/php/tasks/mod-php.yml, making the memory limit a variable so each PHP project can select its own level. Then I'd add `512M` to the FPUL group_vars and set a default value in the PHP role.